static/js/search-filter.js

```diff
@@ -0,0 +1,5 @@
+$('.moj-filter__options')
+  .find(':button')
```

Comment:
this seems a bit odd - why do we need to manually add an onclick to the button to submit the form?

Reply:
the MOJ component structures the submit outside of the form - no idea why it is done like that, but I'll take a second look. This is what we had to do in court register